data/applicationDocument.json

```diff
@@ -0,0 +1,169 @@
+        "11.9":"[11.9] The circumstances of the current offence suggest that Mr Smith acted ..."
+     },
+     {
+        "RM28":"[RM28] Mr Smith is currently on remand awaiting sentence",
+        "RM35":"[RM35] Placeholder content for additional comments",
+        "RM34":"[RM34] Discuss any concerns with line manager/SPO Increase frequency of reporting Bring forward MAPPA/Consider Level 2 MAPPA professionals meetings  considered if MAPPA not applicable. Consider referral to Approved Premises Consider motivational interviewing/engagement officer.  Joint interview with police OM or other relevant agencies? To raise concerns Referral to substance misues worker for additional support. Warning letters to be employed Recall/Breach if risk not manageable.",
+        "RM33":"[RM33] Undertake sentence planning with OS within 8 or 16 weeks of sentence carats - continue to work with Mr Smith and refer for relevant programmes during any prison sentence imposed. DIP supervise DRR is imposed or provide support post release if conditions/requirements to manage the specific risks.",
+        "RM32":"[RM32] Requires referral to community drug DIP upon  release. To link in with employment/training resources such as College and ETEO. Completion of victim empathy module via 1-1 supervision to build upon awareness of impact of offending.   Level setting to be undertaken by Senior Probation Officer and OMU. Multi Agency Public Protection Panel to be convened if level 2 either at start of supervision or 6 months prior to release",
+        "RM31":"[RM31] State they will have secure accommodation for Mr Smith  and partner on release, although she too is subject to a DRR at present. Supportive wider family.  Has completed previously Short Duration programme and attended one of the support sessions .  Engaged with CARATS in custody.  Complied with voluntary drug tests in custody. Some motivation to desist from alcohol/drug use in the future, this needs to be encouraged with use of community resources. Completed drug and heroin awareness programmes in custody.",
+        "RM30":"[RM30] Probation Officer, Education training and employment Officer, Prison Offender Supervisor",
+        "RM28.1":"[RM28.1] Currently on remand at HMP Wandsworth - Management of case under MAPPA - level not yet set."
+     },
+     {
+        "R8.1.1":"Review 06.10.21:\r\n \r\n         There have been numerous ACCTs opened since 2013 and every subsequent year he has been in custody.  In 2021...",
+        "R8.2.1":"Has told prison staff that he swallowed batteries on one occasion due to wanting..",
+        "R8.3.1":"Review 06.10.21:\r\n \r\n         A previous assessor opined that Mr Smith was displaying all the characteristics of someone..."
+     }
+  ],
+  "risk-management-features":[
+     {
+        "Describe why an AP placement is needed to manage the risk of Justyn Flatley":"Some words",
```

Comment:
Changing "Justyn Flatley" to "Mr Smith" (as per the OAsys info) would probably make the experience for testers smoother?

Reply:
That's a good plan - can do that pretty easily I think - same with the other names 👍